vendor/github.com/moby/buildkit/go.mod

```diff
@@ -5,11 +5,11 @@ go 1.13
 require (
 	github.com/AkihiroSuda/containerd-fuse-overlayfs v1.0.0
 	github.com/BurntSushi/toml v0.3.1
-	github.com/Microsoft/go-winio v0.4.15-0.20200908182639-5b44b70ab3ab
-	github.com/Microsoft/hcsshim v0.8.9
+	github.com/Microsoft/go-winio v0.4.15
+	github.com/Microsoft/hcsshim v0.8.10
```

Comment:
I'm unsure about what to do with this dependency; I didn't update because we currently vendor the special "moby" branch, but with both containerd, buildkit, and docker using the dependency, I wonder if we should move back to these tags

https://github.com/Microsoft/hcsshim/compare/9dcb42f100215f8d375b4a9265e5bba009217a85..v0.8.10

Reply:
Opened https://github.com/moby/moby/pull/41689 for discussion